src/Reducers/notesActions-reducer.jsx

```diff
@@ -0,0 +1,16 @@
+
+const notesReducer = (state, action) => {
+    
+    switch (action.type) {
+        case "ADD_NOTE":
+            return {...state, notes : [...action.payload] };
+        
+        case "DELETE_NOTE":
+            return {...state, notes : [...action.payload] };
```

Comment:
Over here you can just write :- `return {...state, notes : action.payload };` and that would also work, why to destructure unnecessarily.